.circleci/src/workflows/release.yml

```diff
@@ -132,10 +126,7 @@ jobs:
       context: github
       requires:
         - release-audius-docker-compose-trigger
-
-  - release-discord-trigger:
-      type: approval
   - release-discord:
       context: discord
       requires:
-        - release-discord-trigger
+        - release-audius-docker-compose-trigger
```

Comment:
Just noting that these means it will be posted daily.

Reply:
Yeah - if it's too noisy we can scale it back. Generally i just don't like having jobs sit here that we never run (and decay to the point of not working). Maybe we could do a different ritual on monthy deploys with minor versions soon...